### DISCUSSION TITLE
Is it possible to implement a fake dual-stage trigger?

### DISCUSSION THREAD

**Discussion by mkaito** [AntiMicroX/antimicrox#308] [Q&A]
Hi!
In playing Final Fantasy XIV, the triggers are held for one action, and double tapped for another. I would like to remap these two actions to a partial and full depression of the same trigger, respectively.
Is this something I can do? If so, how?

**Comment by zzpxyx** [comment]
I don't think AntiMicroX can remap controller actions like these directly. An idea would be to set the game to use keyboard and mouse, and then use AntiMicroX to map the controller to keyboard and mouse. That said, I don't have a testing environment at hand so I'm not sure how feasible that is, especially on whether it's possible to distinguish between partial and full depression of the trigger in AntiMicroX.
